files/image_config/platform/rc.local

```diff
@@ -183,8 +183,32 @@ for x in "$@"; do
 done
 }
 
+program_baud_rate()
+{
+    if [ -n "$aboot_platform" ]; then
+        platform=$aboot_platform
+    elif [ -n "$onie_platform" ]; then
+        platform=$onie_platform
+    else
+        platform=''
+    fi
+
+    # Default console speed: 9600
+    CONSOLE_SPEED=9600
+
+    CONSOLE_CFG=/usr/share/sonic/device/$platform/installer.conf
+    if [ -f $CONSOLE_CFG ]; then
+        . $CONSOLE_CFG
+    fi
+
+    sed -i "s|\-\-keep\-baud .* %I|\-\-keep\-baud $CONSOLE_SPEED %I|g" /lib/systemd/system/serial-getty@.service
+    systemctl daemon-reload
```

Comment:
what is order between serial-getty and rc.local? Do we need to reload the serial-getty after change its baud rate?

Reply:
From my test. We don't need to restart serial-getty. daemon-reload is all that we need.